### ISSUE TITLE
I can't rename a file that i have downloaded from Sia

### ISSUE THREAD

**Issue by Chehands** [NebulousLabs/Sia-UI#466]
A dialogue box appears informing me that the file is open in siad.exe, even though it is not. I am trying to rename it through windows, not the ui.
Note; if i try again in 5 or 10 minutes, it is likely that i will be able to rename it.

**Comment by lukechampine** [comment]
That is odd. siad should only be using the file while it is being downloaded. After it finishes downloading, it closes the file handle. You can see the closing code [here](https://github.com/NebulousLabs/Sia/blob/15481c5b8b511d57fa4fb9ded8a19677a5509e47/modules/renter/download.go#L209).

Are you trying to rename during the download?

**Comment by DavidVorick** [comment]
Maybe the upload loop has control of the file?

**Comment by Chehands** [comment]
No, these are files that have long finished downloading.  It could be something to do with uploading, these are largish test files that I rename and then re-upload, then download etc although they had also finished uploading by several hours.

**Comment by lukechampine** [comment]
it's possible that they haven't completely finished uploading, i.e. siad is still trying to fill out the rest of their redundancy but is repeatedly failing (e.g. due to an offline or desynchronized host).
